orangewidget/utils/messagewidget.py

```diff
@@ -587,11 +587,8 @@ def is_short(m):
         self.__fulltext = fulltext
         self.setToolTip(self.__styled(self.__defaultStyleSheet, fulltext))
         self.anim.start(QPropertyAnimation.KeepWhenStopped)
+        self.__popuptext = fulltext if messages else ""
```

Comment:
Wouldn't `self.__popuptext = fulltext` be enough because when the `if messages` does not apply then fulltext would be `""` anyway (line 579 above).
In that case, do we really still need `self.__popuptext` if it is the same as `self.__fulltext`?

Reply:
I've just updated the PR with this fixed. Thanks.